EIPS/eip-721.md

```diff
@@ -155,22 +155,21 @@ interface ERC165 {
 A wallet/broker/auction application MUST implement the **wallet interface** if it will accept safe transfers.
 
 ```solidity
-/// @dev Note: the ERC-165 identifier for this interface is 0xf0b9e5ba
-/// @dev Note: the application will get the prior owner of the token
-///  via _from parameter -- but it will NOT see who called safeTransferFrom.
+/// @dev Note: the ERC-165 identifier for this interface is 0x150b7a02
```

Comment:
Add a period at end of sentence please. See https://github.com/ethereum/EIPs/pull/1163